ci/qa/phpcpd

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-cd $(dirname $0)/../../
-
-# https://github.com/sebastianbergmann/phpcpd
-vendor/bin/phpcpd ./src
```

Comment:
See my suggestion on the other bundles; there is a substitution for Sebastians abandoned package

Reply:
The shim is the phar 'binary' of the old phpcpd. This solves the composer requirements issue. 
For now, I've used the shim.